oneview/resource_server_profile.go

```diff
@@ -40,6 +40,10 @@ func resourceServerProfile() *schema.Resource {
 				Type:     schema.TypeString,
 				Required: true,
 			},
+			"force": {
```

Comment:
You may want to change this to allow the user to give different force value  like ignore san warning etc as done in case of golang PR

Reply:
waiting for https://github.com/HewlettPackard/oneview-golang/pull/403 to be merged, then I update that PR